CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -536,6 +534,13 @@ local function Create3DMenuItem()
 		self:SetHoverText(hoverTextLabel.Text)
 	end
 
+	local defaultClickBehavior = backgroundButton.MouseButton1Click:connect(function()
```

Comment:
I typically like to name these with `Connection` or `Conn` at the end of the variable name to signify that it is a connection